src/test/serialization_tests.cpp

```diff
@@ -81,8 +81,6 @@ const std::vector<ovms::Precision> UNSUPPORTED_OUTPUT_PRECISIONS{
     // ovms::Precision::BIN, // Cannot create tensor with such precision
     ovms::Precision::BOOL
     // ovms::Precision::CUSTOM),
-
-    // TODO: There are new API 2.0 precisions we do not support. Add tests for those.
```

Comment:
We should check why `UNSUPPORTED_OUTPUT_PRECISIONS` does not contain more unsupported precisions

Reply:
We get:
```
unknown file: Failure
C++ exception with description "cannot locate blob for precision: UNSPECIFIED" thrown in the test body.
```
when we call Tensor constructor. The reason is that in OV 2.0 API there are no such precisions.